kvmagent/kvmagent/plugins/ovn.py

```
@@ -173,21 +173,19 @@ def start_ovn_service(self, req):
-                if dpdkNic.name == nicName:
+                if dpdkNic.pciAddress == pciAddress:
                     found = True
                     driver = dpdkNic.driver
-                    pciAddress = dpdkNic.pciAddress
                     targetDpdkNic.append(dpdkNic)
                     break
 
             if not found:
                 rsp.success = False
-                rsp.error = "nic %s is not found by dpdk-devbind.py"
+                rsp.error = "nic [pci address: %s] is not found by dpdk-devbind.py".format(pciAddress)
```

Ruff (0.8.2)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)